src/Files.App/UserControls/Settings/SettingsDisplayControl.xaml.cs

```diff
@@ -7,69 +7,53 @@ namespace Files.App.UserControls.Settings
 	[ContentProperty(Name = nameof(SettingsActionableElement))]
 	public sealed partial class SettingsDisplayControl : UserControl
 	{
-		public FrameworkElement SettingsActionableElement { get; set; }
```

Comment:
Can you revert the changes to this file?

Reply:
It's done.